schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE IF NOT EXISTS animals (
+    id INT,
```

Comment:
- [ ] (OPTIONAL) Great job so far 👍 it would be great if you could make the `id` column not null and autoincremented and also mark the `id` column as the primary key 💯

Reply:
Thank you @Kingstalux  for this review.
I will make sure to update the ID with the ALTER statement in the next milestone.